tests/foreman/ui/test_activationkey.py

```diff
@@ -379,15 +371,17 @@ def test_negative_create_activation_key_2(self):
         @BZ: 1177158
 
         """
-
-        name = gen_string("alpha", 10)
-        description = gen_string("alpha", 1001)
+        name = gen_string('alpha', 10)
+        description = gen_string('alpha', 1001)
         with Session(self.browser) as session:
-            make_activationkey(session, org=self.org_name,
-                               name=name, env=ENVIRONMENT,
-                               description=description)
-            self.assertIsNotNone(self.activationkey.wait_until_element
-                                 (common_locators["common_haserror"]))
+            make_activationkey(
+                session, org=self.org_name, name=name, env=ENVIRONMENT,
+                description=description)
+            self.assertIsNotNone(
+                self.activationkey.wait_until_element(
+                    common_locators['common_haserror']
+                )
```

Comment:
Please, don't consider as a nitpick, but you used here two different styles for ```wait_until_element``` and ```make_activationkey``` for closing bracket. I'm wondering what style is preferred and where. Or they both are totally equivalent?